staging/kube-oidc-proxy/templates/secret_config.yaml

```diff
@@ -4,14 +4,22 @@ data:
   oidc.ca-pem: {{ .Values.oidc.caPEM | default "" | b64enc }}
   {{ end }}
   oidc.issuer-url: {{ .Values.oidc.issuerUrl | b64enc }}
-  oidc.username-claim: {{ .Values.oidc.usernameClaim | b64enc }}
+  {{ if .Values.oidc.usernameClaim }}
+  oidc.username-claim: {{ .Values.oidc.usernameClaim | default "" | b64enc }}
+  {{ end }}
   oidc.client-id: {{ .Values.oidc.clientId | b64enc }}
+  {{ if .Values.oidc.usernamePrefix }}
   oidc.username-prefix: {{ .Values.oidc.usernamePrefix | default "" | b64enc }}
+  {{ end }}
   oidc.groups-claim: {{ .Values.oidc.groupsClaim | default "" | b64enc }}
   oidc.groups-prefix: {{ .Values.oidc.groupsPrefix | default "" | b64enc }}
   oidc.signing-algs: {{ join "," .Values.oidc.signingAlgs | default "" | b64enc }}
```

Comment:
These don't have possibility of being nil? Since this PR is being defensive should we include these 3?

Reply:
yeah i can add those. Not sure what generally would be empty.